protostar/configuration_file/configuration_file_factory.py

```diff
@@ -79,12 +79,12 @@ def _create_configuration_toml_v1(
 
     def _search_upwards_protostar_toml_path(self) -> Optional[Path]:
         directory_path = self._cwd
-        assert directory_path.exists()
         root_path = Path(directory_path.root)
         while directory_path != root_path:
-            for file_path in directory_path.iterdir():
-                if "protostar.toml" == file_path.name:
-                    return directory_path / "protostar.toml"
+            if directory_path.exists():
```

Comment:
I think we should throw if it doesn't exist

Reply:
Removed the check